spec/lucky/text_helpers/pluralize_spec.cr

```diff
@@ -0,0 +1,24 @@
+require "./text_helpers_spec"
+
+describe Lucky::TextHelpers do
+  describe "pluralize" do
+    it "pluralizes words" do
```

Comment:
Thanks for separating these specs. Great idea on putting them in `text_helpers` 👍